### DISCUSSION TITLE
[Feature Request] Text alignment classes in tooltips

### DISCUSSION THREAD

**Discussion by thdoan** [saadeghi/daisyui#1556] [Ideas / feature requests]
It would be nice to be able to set the text alignment in tooltips using a class like `tooltip-text-left`. I currently have to override the CSS in order to left-align the text in the tooltip:

```
.tooltip:before {
  text-align: left;
}
```

**Comment by saadeghi** [comment]
I don't want to add a new class name just for text alignment. However if you update to the latest version (`2.47.1`) you can use a class name like `before:text-left` to align it anyway you want (or apply other styles)  
Example: https://play.tailwindcss.com/VdrydZQtLz

**Reply by thdoan** [reply]
Cool, thanks for the tip.
